app/templates/view_service.html

```diff
@@ -89,6 +89,7 @@
         {% endcall %}
       {% endfor %}
 
+      {% if current_user.has_role('admin') %}
```

Comment:
The `compare_revisions.html` template could also have this check around the "Update status" button to hide it.

![screen shot 2015-08-25 at 11 05 31](https://cloud.githubusercontent.com/assets/6525554/9464128/3e118620-4b19-11e5-9905-a7dab3c15319.png)